tests/wandb_run_test.py

```diff
@@ -163,3 +163,26 @@ def test_except_hook(test_settings):
     assert "".join(stderr) == "Exception: After wandb.init()\n"
 
     sys.stderr.write = old_stderr_write
+
+
+def assertion(run_id, found, stderr):
+    msg = f"`resume` will be ignored since W&B syncing is set to `offline`. Starting a new run with run id {run_id}"
+    return msg in stderr if found else msg not in stderr
+
+
+@pytest.mark.parametrize(
+    "resume, found",
+    [
+        ("auto", True),
+        ("allow", True),
+        ("never", True),
+        ("must", True),
+        ("", True),
```

Comment:
empty string warns?  this is unexpected isnt it?

Reply:
yep, seems to be due to[ this piece of code](https://github.com/wandb/client/blob/master/wandb/sdk/wandb_settings.py#L1057-L1063).